_pages/fellowship_general.md

```diff
@@ -3,6 +3,38 @@ layout: people
 permalink: /fellowship_general.html
 title: Fellowship General Information
 ---
+<h3 style>NHS Recruitment Regions and AI Project Host Trusts</h3>
+
+* England
+  * East of England
+    * Cambridge University Hospitals
+  * London
+    * Guy's and St Thomas's
```

Comment:
Guy's and St Thomas'

Reply:
Both above changes made- I have moved the regional information down the page in both cases as it's a secondary consideration compared to the other information on the page